ui_framework/doc_site/src/views/accessibility/screen_reader.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+export default () => (
+  <div>
+    <p>
+      This is the first paragraph. It is visible to all.
+    </p>
+    <p className='kuiScreenReaderOnly'>
```

Comment:
There is a cool trick where we can make a component that clones whatever is passed into it and tweaks it slightly. In this case we just want to add a class to that child:

```jsx
import React, {
  cloneElement,
  PropTypes,
} from 'react';
import classNames from 'classnames';

export const KuiScreenReaderOnly = ({ children }) => {
  const classes = classNames('kuiScreenReaderOnly', children.props.className);

  const props = Object.assign({}, children.props, {
    className: classes,
  };

  return cloneElement(children, props);
};

KuiScreenReaderOnly.propTypes = {
  children: PropTypes.node,
};
```

We'd use the component like this:

```jsx
<KuiScreenReaderOnly>
  <p>This is the second paragraph. It is hidden for sighted users but visible to screen readers.</p>
</KuiScreenReaderOnly>
```

We're doing something similar with our typography components for K7. Do you want to give this a shot in this branch? Or if you prefer we can merge it as-is and make this change separately.